src/Compilers/Core/Portable/CodeGen/StateMachineStates.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.CodeGen;
+
+internal static class StateMachineStates
+{
+    internal const int FirstIteratorFinalizeState = -3;
+
+    internal const int FinishedStateMachine = -2;
```

Comment:
>Machine [](http://example.com/codeflow?start=36&length=7)

Should this be "FinishedState" rather than "FinishedStateMachine"? Same question for "NotStartedStateMachine".

Reply:
Renamed.